dbeam-core/src/test/java/com/spotify/dbeam/args/JdbcExportOptionsTest.java

```diff
@@ -23,18 +23,36 @@
 import com.spotify.dbeam.options.JdbcExportArgsFactory;
 import com.spotify.dbeam.options.JdbcExportPipelineOptions;
 
+import java.io.File;
 import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
 import java.time.Period;
 import java.util.Optional;
 
 import org.apache.avro.file.CodecFactory;
 import org.apache.beam.sdk.options.PipelineOptions;
 import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.junit.AfterClass;
 import org.junit.Assert;
+import org.junit.BeforeClass;
 import org.junit.Test;
 
 
 public class JdbcExportOptionsTest {
+  private static File sqlFile;
+
+  @BeforeClass
+  public static void beforeAll() throws IOException {
+    sqlFile = File.createTempFile("query", ".sql");
```

Comment:
I just wonder why there are two different approaches to create temp files.
>sqlFile = File.createTempFile("query", ".sql");
vs
>private static File SQL_FILE = new File(DIR.getAbsolutePath() + ".sql");

Reply:
I found the same between PASSWORD_FILE in JdbcAvroJobTest and  passwordFile in PasswordReaderTest. Better to have one single way, should we  keep create File.createTempFile ?

Reply:
I don't really have an opinion which one is better. I think it is nice to select one and refactor it to a separate method in some test helper class.

Reply:
agree, do you want to do it in the same PR ?

Reply:
I think `File.createTempFile` feels nice!

I don't think it is necessary to address it in this PR. Though it is something for us to follow up.